### ISSUE TITLE
Some newer projects not visible to iPhone users?

### ISSUE THREAD

**Issue by carolboston2** [naturenet/naturenet-www#120]
I created two projects on the website:

Planting for Pollinators and
UMD Stormwater Management Practices

that Liz and Jenny cannot see or contribute to on their iPhones.

I am able to see them and contribute to them on my Android phone.

**Comment by stevemacn** [comment]
@carolboston2 is this a problem with the apps or the website? Both appear to be showing up on the website. I can forward it to the iOS app if this is only affecting that app.

**Comment by wambeere** [comment]
Alright—
So, here is my report:
1.  Still don’t see Carol’s project, “planting for pollinators’ on my iPhone 6. This is true whether I search or scroll down.  (The search function is getting stuck on where you are supposed to choose your community. See the screenshot below.   
2. I do see Carol’s project  on Chrome (macbook pro).  
3. I checked and all my location services on iPhone and on Mac have been on the whole time and in fact NN is the only app with that special ‘allow’ status.
4. Carol had previously mentioned that she could see her project. She uses a non-iPhone product, not sure which one. 
![img_0001](https://user-images.githubusercontent.com/22083448/32749537-abfdecc0-c88d-11e7-80b6-6c87cf973327.JPG)

**Comment by carolboston2** [comment]
Yup--I could always see the projects via PC (Chrome) and on my Android
phone, but Jenny, also using an iPhone, has this problem with my projects,
including the one called

UMD Stormwater Management Practices
